tutormfe/patches/local-docker-compose-dev-services

```diff
@@ -20,9 +20,10 @@
 {%- endfor %}
 
 
-{% if mfe_data.unmounted|length > 0 %}
+{% if mfe_data.unmounted|length > 0 or PARAGON_VERSION is defined %}
```

Comment:
Could you explain where `PARAGON_VERSION` is being defined?

Reply:
I think that having this functionality (the ability to tell this service that it should still host things even if we don't have any MFE bundles to host) is good functionality to add, and doesn't need to be couple to Paragon itself. I think changing this to be something like `STILL_HOST_THINGS_PLEASE` and having the `tutor-contrib-paragon` plugin set that to true when we want to host styles would be ideal.